### ISSUE TITLE
timeout error when taking offer

### ISSUE THREAD

**Issue by pillowtrader** [bisq-network/bisq#2771]
An error occurred when taking the offer.

A timeout occurred.

No funds have left your wallet yet.
Please try to restart your application and check your network connection to see if you can resolve the issue.Please try to restart your application and check your network connection to see if you can resolve the issue.

**Comment by lukeemmett** [comment]
This just happened to me too. 

An error occurred when taking the offer.

A timeout occurred.

No funds have left your wallet yet.
Please try to restart your application and check your network connection to see if you can resolve the issue.Please try to restart your application and check your network connection to see if you can resolve the issue.

**Comment by ghost** [comment]
... so please try to restart your application and check your network connection to see if it's enough to resolve the issue.

**Comment by gamaanderson** [comment]
I had the same issue this morning, i tried everything: restart, update. Nothing helped. I Eventually just gave up

**Comment by ghost** [comment]
Please upload your logfile or the last relevant 100 lines here.
It may help to see what happens.

**Comment by pillowtrader** [comment]
here are the latest lines from my logfile: 

################################################################# 
Apr-23 19:38:51.078 [JavaFX Application Thread] INFO  b.n.p.p.g.RequestDataHandler: Start processing 6 items. 
Apr-23 19:38:51.086 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:38:51.087 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:38:51.087 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:38:51.087 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:38:51.087 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade UMBRA3-8834f5e8-fb11-4ceb-887f-92802d232052-097 
Apr-23 19:38:51.088 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade 88628-6608e613-46c5-445d-b7ea-9e222b2dfd79-101 
Apr-23 19:38:51.088 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade 88628-6608e613-46c5-445d-b7ea-9e222b2dfd79-101 
Apr-23 19:38:51.088 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade 88628-6608e613-46c5-445d-b7ea-9e222b2dfd79-101 
Apr-23 19:38:51.090 [JavaFX Application Thread] INFO  b.c.d.n.l.LiteNode: startParseBlocks 
Apr-23 19:38:51.090 [JavaFX Application Thread] INFO  b.c.d.n.BsqNode: Start parse blocks:
   Start block height=572861
   Genesis txId=4b5417ec5ab6112bedf539c3b4f5a806ed539542d8b717e1c4470aa3180edce5
   Genesis block height=571747
   Block height=572860
 
Apr-23 19:38:51.092 [JavaFX Application Thread] INFO  b.c.d.n.l.n.LiteNodeNetworkService: requestBlocks with startBlockHeight=572861 from peer rm7b56wbrcczpjvl.onion:8000 
Apr-23 19:38:51.093 [JavaFX Application Thread] INFO  b.c.d.n.l.n.RequestBlocksHandler: We request blocks from peer rm7b56wbrcczpjvl.onion:8000 from block height 572861. 
Apr-23 19:38:51.101 [JavaFX Application Thread] INFO  b.c.d.n.l.n.RequestBlocksHandler: Sending of GetBlocksRequest message to peer rm7b56wbrcczpjvl.onion succeeded. 
Apr-23 19:38:52.042 [NetworkNode:SendMessage-to-kwzjoqkcfpdqtzly.onion:9999] INFO  b.n.p.n.NetworkNode: onFailure at sendMessage: peersNodeAddress=kwzjoqkcfpdqtzly.onion:9999
	message=GetPeersRequest
	throwable=java.io.IOException: Cannot connect to hidden service 
Apr-23 19:38:54.706 [NetworkNode:SendMessage-to-dxjam4x7fjsj2lwp.onion:9999] INFO  b.n.p.n.NetworkNode: onFailure at sendMessage: peersNodeAddress=dxjam4x7fjsj2lwp.onion:9999
	message=GetPeersRequest
	throwable=java.io.IOException: Cannot connect to hidden service 
Apr-23 19:38:56.715 [JavaFX Application Thread] INFO  b.n.p.p.PeerManager: We have 11 connections open. Our limit is 12 
Apr-23 19:39:02.332 [JavaFX Application Thread] INFO  b.n.p.p.g.RequestDataHandler: 
#################################################################
Connected to node: ef5qnzx6znifo3df.onion:8000
Received 659 instances
Filter: 1
MailboxStoragePayload: 371
Alert: 1
Arbitrator: 2
OfferPayload: 284
################################################################# 
Apr-23 19:39:02.661 [JavaFX Application Thread] INFO  b.n.p.p.g.RequestDataHandler: Start processing 0 items. 
Apr-23 19:39:04.467 [JavaFX Application Thread] INFO  b.c.d.n.l.n.RequestBlocksHandler: We received from peer rm7b56wbrcczpjvl.onion:8000 a BlocksResponse with 42 blocks 
Apr-23 19:39:04.467 [JavaFX Application Thread] INFO  b.c.d.n.l.n.LiteNodeNetworkService: requestBlocksHandler of outbound connection complete. nodeAddress=rm7b56wbrcczpjvl.onion:8000 
Apr-23 19:39:04.468 [JavaFX Application Thread] INFO  b.c.d.n.l.LiteNode: We received blocks from height 572861 to 572902 
Apr-23 19:39:04.480 [JavaFX Application Thread] INFO  b.c.d.n.p.BlockParser: Parsing 1 transactions at block height 572861 took 7 ms 
Apr-23 19:39:04.581 [JavaFX Application Thread] INFO  b.c.d.n.p.BlockParser: Parsing 3 transactions at block height 572862 took 1 ms 
Apr-23 19:39:04.780 [JavaFX Application Thread] INFO  b.c.d.n.p.BlockParser: Parsing 3 transactions at block height 572867 took 1 ms 
Apr-23 19:39:04.856 [JavaFX Application Thread] INFO  b.c.d.n.p.BlockParser: Parsing 1 transactions at block height 572871 took 1 ms 
Apr-23 19:39:04.889 [JavaFX Application Thread] INFO  b.c.d.n.p.BlockParser: Parsing 0 transactions at block height 572873 took 1 ms 
Apr-23 19:39:05.058 [JavaFX Application Thread] INFO  b.c.d.s.DaoStateSnapshotService: Cloned new snapshotCandidate at height 572880 took 35 ms 
Apr-23 19:39:05.328 [JavaFX Application Thread] INFO  b.c.d.n.p.BlockParser: Parsing 1 transactions at block height 572899 took 1 ms 
Apr-23 19:39:05.363 [JavaFX Application Thread] INFO  b.c.d.s.DaoStateSnapshotService: Saved snapshotCandidate with height 572880 to Disc at height 572900 took 19 ms 
Apr-23 19:39:05.383 [JavaFX Application Thread] INFO  b.c.d.s.DaoStateSnapshotService: Cloned new snapshotCandidate at height 572900 took 19 ms 
Apr-23 19:39:05.476 [JavaFX Application Thread] INFO  b.c.d.n.l.LiteNode: Parsing 42 blocks took 1.008 seconds. 
Apr-23 19:39:05.477 [JavaFX Application Thread] INFO  b.c.d.n.BsqNode: onParseBlockChainComplete 
Apr-23 19:39:05.477 [JavaFX Application Thread] INFO  b.c.d.s.DaoStateService: Parse blockchain completed 
Apr-23 19:39:05.574 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetDaoStateHashesRequest message to peer ef5qnzx6znifo3df.onion:8000 succeeded. 
Apr-23 19:39:05.575 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetDaoStateHashesRequest message to peer rm7b56wbrcczpjvl.onion:8000 succeeded. 
Apr-23 19:39:05.576 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetDaoStateHashesRequest message to peer 5quyxpxheyvzmb2d.onion:8000 succeeded. 
Apr-23 19:39:05.699 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetProposalStateHashesRequest message to peer rm7b56wbrcczpjvl.onion:8000 succeeded. 
Apr-23 19:39:05.699 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetProposalStateHashesRequest message to peer ef5qnzx6znifo3df.onion:8000 succeeded. 
Apr-23 19:39:05.700 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetProposalStateHashesRequest message to peer 5quyxpxheyvzmb2d.onion:8000 succeeded. 
Apr-23 19:39:05.700 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetBlindVoteStateHashesRequest message to peer 5quyxpxheyvzmb2d.onion:8000 succeeded. 
Apr-23 19:39:05.700 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetBlindVoteStateHashesRequest message to peer rm7b56wbrcczpjvl.onion:8000 succeeded. 
Apr-23 19:39:05.701 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: Sending of GetBlindVoteStateHashesRequest message to peer ef5qnzx6znifo3df.onion:8000 succeeded. 
Apr-23 19:39:05.828 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer 5quyxpxheyvzmb2d.onion:8000 a GetDaoStateHashesResponse with 11 stateHashes 
Apr-23 19:39:05.931 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer 5quyxpxheyvzmb2d.onion:8000 a GetProposalStateHashesResponse with 0 stateHashes 
Apr-23 19:39:05.934 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer 5quyxpxheyvzmb2d.onion:8000 a GetBlindVoteStateHashesResponse with 0 stateHashes 
Apr-23 19:39:06.083 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer rm7b56wbrcczpjvl.onion:8000 a GetDaoStateHashesResponse with 11 stateHashes 
Apr-23 19:39:06.085 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer rm7b56wbrcczpjvl.onion:8000 a GetProposalStateHashesResponse with 0 stateHashes 
Apr-23 19:39:06.118 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer ef5qnzx6znifo3df.onion:8000 a GetDaoStateHashesResponse with 11 stateHashes 
Apr-23 19:39:06.163 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer rm7b56wbrcczpjvl.onion:8000 a GetBlindVoteStateHashesResponse with 0 stateHashes 
Apr-23 19:39:06.235 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer ef5qnzx6znifo3df.onion:8000 a GetProposalStateHashesResponse with 0 stateHashes 
Apr-23 19:39:06.680 [JavaFX Application Thread] INFO  b.c.d.m.n.RequestStateHashesHandler: We received from peer ef5qnzx6znifo3df.onion:8000 a GetBlindVoteStateHashesResponse with 0 stateHashes 
Apr-23 19:39:24.798 [JavaFX Application Thread] INFO  b.c.p.p.PriceFeedService: request from provider http://5bmpx76qllutpcyp.onion/ 61.012 sec. after last request 
Apr-23 19:39:26.034 [JavaFX Application Thread] INFO  b.c.p.p.PriceFeedService: Received new MarketPrice(currencyCode=EUR, price=5012.9, timestampSec=1556041281422, isExternallyProvidedPrice=true) from provider http://44mgyoe2b6oqiytt.onion/ after 1.236 sec. 
Apr-23 19:39:31.326 [JavaFX Application Thread] INFO  b.d.m.o.t.TakeOfferDataModel: Start requestTxFee: txFeeFromFeeService=29440 
Apr-23 19:39:31.337 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: SendOfferAvailabilityRequest 
Apr-23 19:39:31.338 [JavaFX Application Thread] INFO  b.c.o.a.t.SendOfferAvailabilityRequest: Send OfferAvailabilityRequest with offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 and uid ca95e10f-43a2-49fb-ab00-91975baac2f0 to peer fiywgoagqa37vm5r.onion:9999 
Apr-23 19:39:31.535 [JavaFX Application Thread] INFO  b.d.m.o.t.TakeOfferDataModel: Completed requestTxFee: txFeeFromFeeService=29440 
Apr-23 19:39:34.566 [JavaFX Application Thread] INFO  b.c.o.a.t.SendOfferAvailabilityRequest: OfferAvailabilityRequest arrived at peer: offerId=fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101; uid=ca95e10f-43a2-49fb-ab00-91975baac2f0 
Apr-23 19:39:35.344 [JavaFX Application Thread] INFO  b.c.o.a.OfferAvailabilityProtocol: Received handleOfferAvailabilityResponse from fiywgoagqa37vm5r.onion:9999 with offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 and uid 3a63a647-172e-4cd4-9c8e-10aad329028f 
Apr-23 19:39:35.346 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: ProcessOfferAvailabilityResponse 
Apr-23 19:39:35.347 [JavaFX Application Thread] INFO  b.c.o.a.OfferAvailabilityProtocol: Send AckMessage for OfferAvailabilityResponse to peer fiywgoagqa37vm5r.onion:9999 with offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 and sourceUid 3a63a647-172e-4cd4-9c8e-10aad329028f 
Apr-23 19:39:35.348 [JavaFX Application Thread] INFO  b.n.p.p.PeerManager: Num of latestLivePeers=9 
Apr-23 19:39:35.349 [JavaFX Application Thread] ERROR b.c.o.a.OfferAvailabilityProtocol: AckMessage for OfferAvailabilityResponse failed. AckMessage=AckMessage{
     uid='f0551160-a6c5-4f1b-8bb5-093ffd44b0a3',
     senderNodeAddress=2x5vqf2fozb2u2bm.onion:9999,
     sourceType=OFFER_MESSAGE,
     sourceMsgClassName='OfferAvailabilityResponse',
     sourceUid='3a63a647-172e-4cd4-9c8e-10aad329028f',
     sourceId='fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101',
     success=true,
     errorMessage='null'
} NetworkEnvelope{
     messageVersion=10
}, makersNodeAddress=fiywgoagqa37vm5r.onion:9999, errorMessage=We did not send the EncryptedMessage because the peer does not support the capability. 
Apr-23 19:39:37.256 [JavaFX Application Thread] INFO  o.b.w.Wallet: Completing send tx with 2 outputs totalling 0.003896 BTC and a fee of 0.00 BTC/kB 
Apr-23 19:39:37.258 [JavaFX Application Thread] INFO  o.b.w.Wallet:   with 0.0158648 BTC change 
Apr-23 19:39:37.273 [JavaFX Application Thread] INFO  o.b.w.Wallet:   with a fee of 0.00092 BTC/kB, 0.0002392 BTC for 260 bytes 
Apr-23 19:39:37.273 [JavaFX Application Thread] INFO  o.b.w.Wallet:   completed:   4ffaec47303e08d84603aa9237c04e78e89fd81d1ff2d1a20188e562e982727c
     in   PUSHDATA(72)[30450221009f779d192493124ff33a3a20835ebf783793b247a9ba995718d4ad0409458c62022012d24d6112d080eb62fb78102d8cb91857e8927b27932b15ea4a581cf74901bd01] PUSHDATA(33)[030a6e4a498f4eb87d052297fd572e10614ecec6f0915f9fca67fdb6089a00107b] 0.02 BTC (2000000)
          outpoint:652a5c4538b0369b5626077a84651ed995e8fe634afe953130f27d13ebb36b31:0 hash160:1c01c272d540a64fa26d1379b45c16f7cfdd4c7f
     out  DUP HASH160 PUSHDATA(20)[22c84c29e763809ee109cbc23cd9af715ec1b66d] EQUALVERIFY CHECKSIG 0.0001872 BTC (18720) ScriptPubKey: 76a91422c84c29e763809ee109cbc23cd9af715ec1b66d88ac Address:14AuvSNoQ1ASeCDxUsfL9SZvzC2nBAG1Zm 
     out  DUP HASH160 PUSHDATA(20)[22c84c29e763809ee109cbc23cd9af715ec1b66d] EQUALVERIFY CHECKSIG 0.0037088 BTC (370880) ScriptPubKey: 76a91422c84c29e763809ee109cbc23cd9af715ec1b66d88ac Address:14AuvSNoQ1ASeCDxUsfL9SZvzC2nBAG1Zm 
     out  DUP HASH160 PUSHDATA(20)[22c84c29e763809ee109cbc23cd9af715ec1b66d] EQUALVERIFY CHECKSIG 0.0158648 BTC (1586480) ScriptPubKey: 76a91422c84c29e763809ee109cbc23cd9af715ec1b66d88ac Address:14AuvSNoQ1ASeCDxUsfL9SZvzC2nBAG1Zm 
     fee  0.0002392 BTC for 260 bytes (92 Satoshi/Byte)
     prps USER_PAYMENT
 
Apr-23 19:39:37.276 [JavaFX Application Thread] INFO  b.c.b.TxFeeEstimationService: Fee estimation resulted in a tx size of 260 bytes.
We use an average between the taker fee tx and the deposit tx (320 bytes) which results in 290 bytes.
The payout tx has 380 bytes, we use that as our min value. Size for fee calculation is 380 bytes.
The tx fee of 34960 Sat 
Apr-23 19:39:44.570 [JavaFX Application Thread] INFO  b.n.p.p.PeerManager: We have 12 connections open. Our limit is 12 
Apr-23 19:39:45.656 [JavaFX Application Thread] INFO  b.n.p.n.Connection: We did not send the message because the peer does not support our required capabilities. message=AddPersistableNetworkPayloadMessage(persistableNetworkPayload=AccountAgeWitness{
     hash=2e25b46168c4fb455198124e0f7eb785ff864efe,
     date=Tue Apr 23 19:39:43 CEST 2019
}), peer=Optional[fiywgoagqa37vm5r.onion:9999], peers supportedCapabilities=[] 
Apr-23 19:39:52.271 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: SendOfferAvailabilityRequest 
Apr-23 19:39:52.271 [JavaFX Application Thread] INFO  b.c.o.a.t.SendOfferAvailabilityRequest: Send OfferAvailabilityRequest with offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 and uid 27f4f184-1809-48a7-9c37-a1123418f0b6 to peer fiywgoagqa37vm5r.onion:9999 
Apr-23 19:39:52.279 [JavaFX Application Thread] INFO  b.c.o.a.t.SendOfferAvailabilityRequest: OfferAvailabilityRequest arrived at peer: offerId=fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101; uid=27f4f184-1809-48a7-9c37-a1123418f0b6 
Apr-23 19:39:52.965 [JavaFX Application Thread] INFO  b.c.o.a.OfferAvailabilityProtocol: Received handleOfferAvailabilityResponse from fiywgoagqa37vm5r.onion:9999 with offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 and uid 499e7c4c-5bb4-497f-b344-d94182bad905 
Apr-23 19:39:52.966 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: ProcessOfferAvailabilityResponse 
Apr-23 19:39:52.966 [JavaFX Application Thread] INFO  b.c.o.a.OfferAvailabilityProtocol: Send AckMessage for OfferAvailabilityResponse to peer fiywgoagqa37vm5r.onion:9999 with offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 and sourceUid 499e7c4c-5bb4-497f-b344-d94182bad905 
Apr-23 19:39:52.967 [JavaFX Application Thread] ERROR b.c.o.a.OfferAvailabilityProtocol: AckMessage for OfferAvailabilityResponse failed. AckMessage=AckMessage{
     uid='e2972cc3-c908-493d-9a25-75803c083aa0',
     senderNodeAddress=2x5vqf2fozb2u2bm.onion:9999,
     sourceType=OFFER_MESSAGE,
     sourceMsgClassName='OfferAvailabilityResponse',
     sourceUid='499e7c4c-5bb4-497f-b344-d94182bad905',
     sourceId='fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101',
     success=true,
     errorMessage='null'
} NetworkEnvelope{
     messageVersion=10
}, makersNodeAddress=fiywgoagqa37vm5r.onion:9999, errorMessage=We did not send the EncryptedMessage because the peer does not support the capability. 
Apr-23 19:39:52.977 [JavaFX Application Thread] INFO  b.c.n.a.TradeEvents: We got a new trade. id=fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 
Apr-23 19:39:52.983 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: TakerSelectMediator 
Apr-23 19:39:52.985 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: TakerVerifyMakerAccount 
Apr-23 19:39:52.985 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: TakerVerifyMakerFeePayment 
Apr-23 19:39:52.985 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: CreateTakerFeeTx 
Apr-23 19:39:53.014 [JavaFX Application Thread] INFO  o.b.w.WalletFiles: Saving wallet; last seen block is height 572902, date 2019-04-23T17:29:04Z, hash 000000000000000000083acfc9c75e11408595fdd2440f2ee63c1aab1900fcba 
Apr-23 19:39:53.021 [JavaFX Application Thread] INFO  o.b.w.WalletFiles: Save completed in 6.662 ms 
Apr-23 19:39:53.023 [JavaFX Application Thread] INFO  o.b.w.WalletFiles: Saving wallet; last seen block is height 572902, date 2019-04-23T17:29:04Z, hash 000000000000000000083acfc9c75e11408595fdd2440f2ee63c1aab1900fcba 
Apr-23 19:39:53.028 [JavaFX Application Thread] INFO  o.b.w.WalletFiles: Save completed in 4.878 ms 
Apr-23 19:39:53.029 [JavaFX Application Thread] INFO  b.c.b.w.TradeWalletService: reservedFundsForOffer 0.0038192 
Apr-23 19:39:53.030 [JavaFX Application Thread] INFO  b.c.b.w.TradeWalletService: tradingFee 0.0001872 
Apr-23 19:39:53.030 [JavaFX Application Thread] INFO  b.c.b.w.TradeWalletService: txFee 0.0003496 
Apr-23 19:39:53.030 [JavaFX Application Thread] INFO  o.b.w.Wallet: Completing send tx with 2 outputs totalling 0.0040064 BTC and a fee of 0.00 BTC/kB 
Apr-23 19:39:53.031 [JavaFX Application Thread] INFO  o.b.w.Wallet:   with 0.015644 BTC change 
Apr-23 19:39:53.033 [JavaFX Application Thread] INFO  o.b.w.Wallet:   with a fee of 0.0013498 BTC/kB, 0.0003496 BTC for 259 bytes 
Apr-23 19:39:53.033 [JavaFX Application Thread] INFO  o.b.w.Wallet:   completed:   cb869e62d32adab57f4be7b00e8408ed0a26923f41fae6d2bcf62e1bac516577
     in   PUSHDATA(71)[3044022005de8329c8e1dc76c718f116032e6b45573b13a2c080a24dd06de87a574a52340220603e151031cfd64a714a455b0a95e99a5bdec5dc2fb693019d5f359d3e0afbed01] PUSHDATA(33)[030a6e4a498f4eb87d052297fd572e10614ecec6f0915f9fca67fdb6089a00107b] 0.02 BTC (2000000)
          outpoint:652a5c4538b0369b5626077a84651ed995e8fe634afe953130f27d13ebb36b31:0 hash160:1c01c272d540a64fa26d1379b45c16f7cfdd4c7f
     out  DUP HASH160 PUSHDATA(20)[f6876dfc59041e46bdaece49428676dd9e5b3226] EQUALVERIFY CHECKSIG 0.0001872 BTC (18720) ScriptPubKey: 76a914f6876dfc59041e46bdaece49428676dd9e5b322688ac Address:1PUXU1MQ82JC3Hx1NN5tZs3BaTAJVg72MC 
     out  DUP HASH160 PUSHDATA(20)[e2007f27e8d8e1642ce643724da939b46089489d] EQUALVERIFY CHECKSIG 0.0038192 BTC (381920) ScriptPubKey: 76a914e2007f27e8d8e1642ce643724da939b46089489d88ac Address:1MbzJsBUy7tXbeFWUE6dPrQqong8WhwMXX 
     out  DUP HASH160 PUSHDATA(20)[c20706312253a37e37b79f12defb339fc6ce4dc6] EQUALVERIFY CHECKSIG 0.015644 BTC (1564400) ScriptPubKey: 76a914c20706312253a37e37b79f12defb339fc6ce4dc688ac Address:1JgvWGKirm3zRj8vjvXubHKeNuNPZ2Puod 
     fee  0.0003496 BTC for 259 bytes (134 Satoshi/Byte)
     prps USER_PAYMENT
 
Apr-23 19:39:53.035 [JavaFX Application Thread] INFO  b.c.b.w.WalletService: 
tradingFeeTx:
  cb869e62d32adab57f4be7b00e8408ed0a26923f41fae6d2bcf62e1bac516577
     in   PUSHDATA(71)[3044022005de8329c8e1dc76c718f116032e6b45573b13a2c080a24dd06de87a574a52340220603e151031cfd64a714a455b0a95e99a5bdec5dc2fb693019d5f359d3e0afbed01] PUSHDATA(33)[030a6e4a498f4eb87d052297fd572e10614ecec6f0915f9fca67fdb6089a00107b] 0.02 BTC (2000000)
          outpoint:652a5c4538b0369b5626077a84651ed995e8fe634afe953130f27d13ebb36b31:0 hash160:1c01c272d540a64fa26d1379b45c16f7cfdd4c7f
     out  DUP HASH160 PUSHDATA(20)[f6876dfc59041e46bdaece49428676dd9e5b3226] EQUALVERIFY CHECKSIG 0.0001872 BTC (18720) ScriptPubKey: 76a914f6876dfc59041e46bdaece49428676dd9e5b322688ac Address:1PUXU1MQ82JC3Hx1NN5tZs3BaTAJVg72MC 
     out  DUP HASH160 PUSHDATA(20)[e2007f27e8d8e1642ce643724da939b46089489d] EQUALVERIFY CHECKSIG 0.0038192 BTC (381920) ScriptPubKey: 76a914e2007f27e8d8e1642ce643724da939b46089489d88ac Address:1MbzJsBUy7tXbeFWUE6dPrQqong8WhwMXX 
     out  DUP HASH160 PUSHDATA(20)[c20706312253a37e37b79f12defb339fc6ce4dc6] EQUALVERIFY CHECKSIG 0.015644 BTC (1564400) ScriptPubKey: 76a914c20706312253a37e37b79f12defb339fc6ce4dc688ac Address:1JgvWGKirm3zRj8vjvXubHKeNuNPZ2Puod 
     fee  0.0003496 BTC for 259 bytes (134 Satoshi/Byte)
     prps USER_PAYMENT
 
Apr-23 19:39:53.035 [JavaFX Application Thread] INFO  b.c.b.w.BtcWalletService: swap addressEntry with address 1LGKjuJaagVJPboJWUULGS51FRfLn6TtVi and offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 from context OFFER_FUNDING to available 
Apr-23 19:39:53.035 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: BuyerAsTakerCreatesDepositTxInputs 
Apr-23 19:39:53.037 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: TakerSendPayDepositRequest 
Apr-23 19:39:53.039 [JavaFX Application Thread] INFO  b.c.t.p.t.t.TakerSendPayDepositRequest: Send PayDepositRequest with offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 and uid 35ecfc41-61d1-4c13-ad01-97444bd42063 to peer fiywgoagqa37vm5r.onion:9999 
Apr-23 19:39:53.139 [JavaFX Application Thread] INFO  b.c.t.p.t.t.TakerSendPayDepositRequest: PayDepositRequest arrived at peer: offerId=fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101; uid=35ecfc41-61d1-4c13-ad01-97444bd42063 
Apr-23 19:40:12.275 [BlockingClient network thread for r3dsojfhwcm7x7p6.onion:8333] ERROR o.b.n.BlockingClient: Error trying to open/read from connection: r3dsojfhwcm7x7p6.onion:8333: SOCKS: TTL expired 
Apr-23 19:40:12.276 [BlockingClient network thread for r3dsojfhwcm7x7p6.onion:8333] INFO  o.b.c.PeerGroup: [r3dsojfhwcm7x7p6.onion]:8333: Peer died      (8 connected, 0 pending, 9 max) 
Apr-23 19:40:12.276 [PeerGroup Thread] INFO  o.b.c.PeerGroup: Waiting 1000 msec before next connect attempt to [lgkvbvro67jomosw.onion]:8333 
Apr-23 19:40:12.289 [AbstractTimeoutHandler timeouts] INFO  o.b.c.PeerSocketHandler: [r3dsojfhwcm7x7p6.onion]:8333: Timed out 
Apr-23 19:40:12.290 [AbstractTimeoutHandler timeouts] INFO  o.b.c.PeerGroup: [r3dsojfhwcm7x7p6.onion]:8333: Peer died      (8 connected, 0 pending, 9 max) 
Apr-23 19:40:12.290 [PeerGroup Thread] INFO  o.b.c.PeerGroup: Waiting 1500 msec before next connect attempt to [sjyzmwwu6diiit3r.onion]:8333 
Apr-23 19:40:13.277 [PeerGroup Thread] INFO  o.b.c.PeerGroup: Waiting 513 msec before next connect attempt to [fz6nsij6jiyuwlsc.onion]:8333 
Apr-23 19:40:13.794 [PeerGroup Thread] INFO  o.b.c.PeerGroup: Attempting connection to [sjyzmwwu6diiit3r.onion]:8333     (8 connected, 1 pending, 9 max) 
Apr-23 19:40:16.148 [BlockingClient network thread for sjyzmwwu6diiit3r.onion:8333] INFO  o.b.core.Peer: Announcing to sjyzmwwu6diiit3r.onion:8333 as: /bitcoinj:0.14.7.bisq.1-SNAPSHOT/Bisq:1.0.1/ 
Apr-23 19:40:16.514 [BlockingClient network thread for sjyzmwwu6diiit3r.onion:8333] INFO  o.b.core.Peer: [sjyzmwwu6diiit3r.onion]:8333: Got version=70015, subVer='/Satoshi:0.17.0/', services=0x1037, time=2019-04-23 19:40:16, blocks=572902 
Apr-23 19:40:16.515 [BlockingClient network thread for sjyzmwwu6diiit3r.onion:8333] INFO  o.b.c.PeerGroup: [sjyzmwwu6diiit3r.onion]:8333: New peer      (9 connected, 0 pending, 9 max) 
Apr-23 19:40:23.086 [JavaFX Application Thread] INFO  b.c.d.n.l.n.LiteNodeNetworkService: We received a new message from peer Optional[5quyxpxheyvzmb2d.onion:8000] and broadcast it to our peers. extBlockId=00000000000000000022a139e0f782fbef2a7a230aeea6f5cc065597aa6ef30a:[] 
Apr-23 19:40:23.088 [JavaFX Application Thread] INFO  b.c.d.n.l.LiteNode: onNewBlockReceived: block at height 572903, hash=00000000000000000022a139e0f782fbef2a7a230aeea6f5cc065597aa6ef30a 
Apr-23 19:40:23.089 [JavaFX Application Thread] INFO  b.c.d.s.DaoStateService: New Block added at blockHeight 572903 
Apr-23 19:40:23.089 [JavaFX Application Thread] INFO  b.c.d.s.DaoStateService: Parse block completed: Block height 572903, 0 BSQ transactions. 
Apr-23 19:40:23.162 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
Apr-23 19:40:23.163 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
Apr-23 19:40:23.346 [JavaFX Application Thread] INFO  b.n.p.n.Connection: We did not send the message because the peer does not support our required capabilities. message=NetworkEnvelope{
     messageVersion=10
}, peer=Optional[xlugoduy3mgh6ud7.onion:9999], peers supportedCapabilities=[0, 1, 2, 7] 
Apr-23 19:40:26.353 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Background saving wallet; last seen block is height 572903, date 2019-04-23T17:40:18Z, hash 00000000000000000022a139e0f782fbef2a7a230aeea6f5cc065597aa6ef30a 
Apr-23 19:40:26.353 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Background saving wallet; last seen block is height 572903, date 2019-04-23T17:40:18Z, hash 00000000000000000022a139e0f782fbef2a7a230aeea6f5cc065597aa6ef30a 
Apr-23 19:40:26.364 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Save completed in 10.61 ms 
Apr-23 19:40:26.364 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Save completed in 10.67 ms 
Apr-23 19:40:27.804 [JavaFX Application Thread] INFO  b.c.p.p.PriceFeedService: request from provider http://5bmpx76qllutpcyp.onion/ 63.006 sec. after last request 
Apr-23 19:40:29.040 [JavaFX Application Thread] INFO  b.c.p.p.PriceFeedService: Received new MarketPrice(currencyCode=EUR, price=5010.84, timestampSec=1556041371428, isExternallyProvidedPrice=true) from provider http://44mgyoe2b6oqiytt.onion/ after 1.235 sec. 
Apr-23 19:40:36.189 [JavaFX Application Thread] INFO  b.n.p.n.Connection: We did not send the message because the peer does not support our required capabilities. message=NewStateHashMessage{
     stateHash=StateHash{
     height=572903,
     hash=32fbd7e6efc99fb203be38f05a5e9b073919a247,
     prevHash=05dd448010bac71b27b5ebbf3a221b631ed92e72
}
} NetworkEnvelope{
     messageVersion=10
}, peer=Optional[xlugoduy3mgh6ud7.onion:9999], peers supportedCapabilities=[0, 1, 2, 7] 
Apr-23 19:40:36.689 [JavaFX Application Thread] INFO  b.n.p.n.Connection: We did not send the message because the peer does not support our required capabilities. message=NewStateHashMessage{
     stateHash=StateHash{
     height=572903,
     hash=32fbd7e6efc99fb203be38f05a5e9b073919a247,
     prevHash=05dd448010bac71b27b5ebbf3a221b631ed92e72
}
} NetworkEnvelope{
     messageVersion=10
}, peer=Optional[sowyti43j4lxqejb.onion:9999], peers supportedCapabilities=[0, 1, 2, 7] 
Apr-23 19:40:36.739 [JavaFX Application Thread] INFO  b.n.p.n.Connection: We did not send the message because the peer does not support our required capabilities. message=NewStateHashMessage{
     stateHash=StateHash{
     height=572903,
     hash=32fbd7e6efc99fb203be38f05a5e9b073919a247,
     prevHash=05dd448010bac71b27b5ebbf3a221b631ed92e72
}
} NetworkEnvelope{
     messageVersion=10
}, peer=Optional[fiywgoagqa37vm5r.onion:9999], peers supportedCapabilities=[] 
Apr-23 19:40:44.274 [NetworkNode:SendMessage-to-rm32roxvhvw3x5go.onion:9999] INFO  b.n.p.n.NetworkNode: onFailure at sendMessage: peersNodeAddress=rm32roxvhvw3x5go.onion:9999
	message=GetPeersRequest
	throwable=java.io.IOException: Cannot connect to hidden service 
Apr-23 19:40:45.275 [NetworkNode:SendMessage-to-vspxlga6yexcroln.onion:9999] INFO  b.n.p.n.NetworkNode: onFailure at sendMessage: peersNodeAddress=vspxlga6yexcroln.onion:9999
	message=GetPeersRequest
	throwable=java.io.IOException: Cannot connect to hidden service 
Apr-23 19:41:22.996 [JavaFX Application Thread] ERROR b.c.t.p.TradeProtocol: Timeout reached. TradeID=fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101, state=PREPARATION 
Apr-23 19:41:23.014 [JavaFX Application Thread] WARN  b.c.t.p.TradeProtocol: cleanupTradableOnFault tradeState=PREPARATION 
Apr-23 19:41:23.015 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade OnKvZCj-f8e924bc-4941-4862-b97b-ec2f97b6bc38-094 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade UMBRA3-8834f5e8-fb11-4ceb-887f-92802d232052-097 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade 88628-6608e613-46c5-445d-b7ea-9e222b2dfd79-101 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade 88628-6608e613-46c5-445d-b7ea-9e222b2dfd79-101 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade 88628-6608e613-46c5-445d-b7ea-9e222b2dfd79-101 
Apr-23 19:41:23.016 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 
Apr-23 19:41:23.016 [JavaFX Application Thread] INFO  b.c.b.w.BtcWalletService: swap addressEntry with address 17Qfpvdds9SGtP72e3tEvLDNhkN1BqnQXm and offerId fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101 from context MULTI_SIG to available 
Apr-23 19:41:23.017 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade fY7FS-0d121e63-e331-4bbd-b3d1-8d9fb770ddc6-101

**Comment by pillowtrader** [comment]
restarting several times has not helped. also rebooting my computer did not resolve the issue. 
This has been happening since yesterday. I thought id give it a days rest, but today its still the same problem.

**Comment by ghost** [comment]
To All
Did you verify if your onion is not falsely blacklisted due to recent scam ?
https://bisq.community/t/urgent-certain-banks-banned-until-further-notice/7420/9

**Comment by pillowtrader** [comment]
@HarryMacfinned yes I have checked the list and my onion is not stated on that list.

**Comment by ripcurlx** [comment]
@pillowtrader Is this happening to you trying to take different offers or always the same? If it is not a problem for you to share: Which country, market and payment method are you trying to take existing offers from?

**Comment by pillowtrader** [comment]
@ripcurlx this is happening on various orders I'm trying to take. I'm trying to buy btc with eur using sepa.  Different orders from different countries, but all within EU.

**Comment by lukeemmett** [comment]
This is still happening to me. Tried rebooting my laptop but that didn't do anything. Checked that list and my onion address isn't listed. I'm trying to purchase BTC using USD with either Chase Quickpay or Zelle and I'm getting this same timeout error for both. Happy to share part of the log file so long as I'm not in the process sharing any private or identifiable info!

**Comment by ghost** [comment]
@lukeemmett 
Please copy/paste from the logfile the very relevant part only, eg the minutes were the issue happens.
Take care yourself about private info.
This will simplify and accelerate the log analysis. (This scam makes everybody busy atm).
(If the whole logfile is needed, we'll ask you).

**Comment by 01mercury** [comment]
I also get a timeout error when taking offer. I working with te latest version of Bisq. Restarting does not help. My connection is using provided bitcoin node,  and thor adres is not part of the blocked list. what to do?

**Comment by ghost** [comment]
@01mercury 
Please post a link to the relevant part of your logfile.

**Comment by 01mercury** [comment]
I'm affraid I do not know what the relevant part is so I attached the log.
[Log bisq.txt](https://github.com/bisq-network/bisq/files/3120887/Log.bisq.txt)

**Comment by ManfredKarrer** [comment]
@01mercury What is your banks country?

**Comment by kozy** [comment]
Same issue of timeout when taking any trade here, last version, Netherlands

**Comment by 01mercury** [comment]
NL

**Comment by ManfredKarrer** [comment]
@01mercury That country is blocked as well atm due to the chargeback scams. With next release we will add more protection tools and then we unblock NL.

**Comment by ManfredKarrer** [comment]
See https://bisq.community/t/urgent-certain-banks-banned-until-further-notice/7420/42

**Comment by 01mercury** [comment]
Any prospects of when that might be?

**Comment by ghost** [comment]
2 weeks should be necessary.

**Comment by dgyg** [comment]
This problem still exists.
```
Jun-10 19:12:04.641 [JavaFX Application Thread] ERROR b.c.o.a.OfferAvailabilityProtocol: AckMessage for OfferAvailabilityResponse failed. AckMessage=AckMessage{
     uid='xxxx-xxx-xx-xx-xxxxxx',
     senderNodeAddress=xxxxxxxx.onion:9999,
     sourceType=OFFER_MESSAGE,
     sourceMsgClassName='OfferAvailabilityResponse',
     sourceUid='xxxxxx-xxx-xxx-xx-xxxxexxxx',
     sourceId='xxxxxxxx-xxxxxxxx-xxxx-xxxxx-xxxx-xxxxx',
     success=true,
     errorMessage='null'
} NetworkEnvelope{
     messageVersion=10
}, makersNodeAddress=xxxxxxxxx.onion:9999, errorMessage=We did not send the EncryptedMessage because the peer does not support the capability. 
.....
Jun-10 19:13:34.648 [JavaFX Application Thread] ERROR b.c.t.p.TradeProtocol: Timeout reached. TradeID=xxxxxxxx-xxxxxxxx-xxxx-xxxxx-xxxx-xxxxx38-110, state=PREPARATION
```
I noticed that for the offer I wanted to take, a previous statement saying

`Jun-10 19:09:46.480 [JavaFX Application Thread] WARN  b.c.t.TradeManager: We found an outdated addressEntry for trade xxxxxxxx-xxxxxxxx-xxxx-xxxxx-xxxx-xxxxx38-110 `

has been logged. 

Could it be that an old client is advertising an offer that can not be taken by the newest client program?
